tests/test_config.py

```diff
@@ -590,6 +590,81 @@ def test_config_precedence():
     assert cfg(f=LOCAL)['pytest']['precedence'] == 'local'
 
 
+def test_config_multiple(config_tmpdir):
+    # Verify that local settings take precedence over global ones,
+    # but that both values are still available, and that setting
+    # either doesn't affect system settings.
+    def write_config(config_file, section, key1, value1, key2, value2):
+        config_file.parent.mkdir(exist_ok=True)
+
+        content = textwrap.dedent(f'''
+        [{section}]
+        {key1} = {value1}
+        {key2} = {value2}
+        ''')
+
+        with open(config_file, 'w') as conf:
+            conf.write(content)
+
+    # config file paths
+    config_dir = pathlib.Path(config_tmpdir) / 'configs'
+    config_s1 = config_dir / 'system1'
+    config_s2 = config_dir / 'system2'
+    config_g1 = config_dir / 'global1'
+    config_g2 = config_dir / 'global2'
+    config_l1 = config_dir / 'local1'
+    config_l2 = config_dir / 'local2'
```

Comment:
```suggestion
    config_l1 = config_dir / 'local* 1'
    config_l2 = config_dir / 'local  2'
```
Etc., see below. Test code must always try to break things.